core/func_ref.cpp

```diff
@@ -92,7 +92,10 @@ void FuncRef::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("call_funcv", "arg_array"), &FuncRef::call_funcv);
 
 	ClassDB::bind_method(D_METHOD("set_instance", "instance"), &FuncRef::set_instance);
+	ClassDB::bind_method(D_METHOD("is_valid"), &FuncRef::is_valid);
+
 	ClassDB::bind_method(D_METHOD("set_function", "name"), &FuncRef::set_function);
 	ClassDB::bind_method(D_METHOD("get_function"), &FuncRef::get_function);
-	ClassDB::bind_method(D_METHOD("is_valid"), &FuncRef::is_valid);
+
+	ADD_PROPERTY(PropertyInfo(Variant::STRING_NAME, "function", PROPERTY_HINT_NONE, "", 0), "set_function", "get_function");
```

Comment:
Is the usage hint `0` necessary for this property? Usually it's used to prevent serialization or showing it in the editor, but I'm not sure this is relevant for a Reference-based class?

Reply:
I thought that was the case, but you are right. We don't need it -- it still shows on the debugger.

This is my first time adding a property so this is new to me.